docs/content/toolbar.md

```diff
@@ -57,20 +57,35 @@ Opens the [Inspector](https://docs.improbable.io/reference/13.1/shared/glossary#
 
 The toolbar settings are in **Edit** > **Project Settings** > **SpatialOS Unreal GDK** > **Toolbar**.
 
- ![Ttoolbar](../assets/screen_grabs/toolbar/toolbar_settings.png)
+ ![Toolbar](../assets/screen_grabs/toolbar/toolbar_settings.png)
 
+* **Configuration**
 
-* **Project Root Folder**</br>
-By default, this points to the root folder of your SpatialOS Unreal GDK project. If you're using a non-standard structure, you'll need to set this yourself.
+    * **Project Root Folder**</br>
+    If you're using a non-standard structure, you'll need to set this yourself. By default, this is empty. If this is left empty, then the it will default to `<ProjectFileLocation>/../spatial`.
```

Comment:
It's not clear who has to change it - making it active voice and present simple tense to aid clarity.

```
This is empty by default. If you leave it empty, it defaults to `<ProjectFileLocation>/../spatial`.
```